modules/currency-l0/src/main/scala/org/tessellation/currency/l0/http/routes/DataBlockRoutes.scala

```diff
@@ -38,4 +41,11 @@ final case class DataBlockRoutes[F[_]: Async](
           case Right(_) => Ok()
         }
   }
+
+  protected val p2p: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "state" / "off-chain" =>
```

Comment:
Using off-chain state might be ambiguous nomenclature, since off-chain is often used to describe data which has a hash reference on chain is stored separately entirely off-chain. A better name might be reflective of the fact that this is information that can only be reconstructed by calculating all operations embedded within the chain data. So it's entirely 'available' within the data data, but only if you reconstruct or recalculate from genesis.

Maybe calculatedState or something reflective of that ? To distinguish it from data which is completely isolated from the chain data?

Reply:
done :+1: